hypertrace-trace-enricher/enriched-span-constants/src/main/proto/org/hypertrace/traceenricher/enrichedspan/constants/v1/enriched_span_attribute.proto

```diff
@@ -95,3 +95,10 @@ enum ErrorMetrics {
     ERROR_METRICS_TOTAL_SPANS_WITH_ERRORS = 3 [(string_value) = "span_count_with_errors"];
     ERROR_METRICS_TOTAL_SPANS_WITH_EXCEPTIONS = 4 [(string_value) = "span_count_with_exceptions"];
 }
+
+enum Deployment {
+    DEPLOYMENT_CANARY = 0 [(string_value) = "canary"];
```

Comment:
just a scenario , can happen for application with canary/baseline workers 

just add a comment in code its not implemented for that yet

Reply:
it can happen (very rare) that a service has the following worker configurations
-worker primary
- worker-canary
- worker-baseline

this is not handled right now, and its an edge case as well
saying to add comment in the code stating this for future reference